src/pages/User/impact/ImpactMissing.tsx

```diff
@@ -28,13 +29,21 @@ const isAllInvisible = (fields, visibleFields) => {
   return false
 }
 
+const isPageOwnerCheck = (activeUser, user) => {
+  if (activeUser && user && toJS(activeUser).userName === user.userName) {
+    return true
+  }
+
+  return false
+}
```

Comment:
I'm happy to approve this but we could make it more succinct I think, maybe it sacrifices some readability though.

```suggestion
const isPageOwnerCheck = (activeUser, user) => activeUser && user && toJS(activeUser).userName === user.userName
```